core/Tracker/Response.php

```diff
@@ -25,7 +25,7 @@ public function init(Tracker $tracker)
     {
         ob_start(); // we use ob_start only because of Common::printDebug, we should actually not really use ob_start
 
-        if ($tracker->isDebugModeEnabled()) {
+        if ($tracker->isDebugModeEnabled() && TrackerConfig::getConfigValue('enable_sql_profiler')) {
```

Comment:
@simivar I'm wondering maybe it can be useful to enable `enable_sql_profiler` without enabling `$tracker->isDebugModeEnabled()`? Not sure... If we make it independent, we would also need to adjust the code in line `90` where we print the profiling results. I suppose it's fine though. Maybe let's just leave it like this for now.